config/schema/elasticsearch_types/international_development_fund.json

```diff
@@ -37,10 +37,6 @@
         "label": "Bangladesh",
         "value": "bangladesh"
       },
-      {
-        "label": "Burma",
-        "value": "burma"
-      },
```

Comment:
As we're now thinking of manually re-tagging the content, it's probably good to avoid removing Burma here until the content is re-tagged.

Reply:
Didn't notice this, thanks!